src/app/main.cpp

```diff
@@ -362,6 +403,94 @@ void displayBadArgMessage(const QString &message)
 #endif
 }
 
+namespace
+{
+#ifndef Q_OS_WIN
+/*
+ * Borrowed from https://stackoverflow.com/a/758054
+ *
+ * getexename - Get the filename of the currently running executable
+ *
+ * The getexename() function copies an absolute filename of the currently
+ * running executable to the array pointed to by buf, which is of length size.
+ *
+ * If the filename would require a buffer longer than size elements, NULL is
+ * returned, and errno is set to ERANGE; an application should check for this
+ * error, and allocate a larger buffer if necessary.
+ *
+ * Return value:
+ * NULL on failure, with errno set accordingly, and buf on success. The
+ * contents of the array pointed to by buf are undefined on error.
+ *
+ * Notes:
+ * This function is tested on Linux only. It relies on information supplied by
+ * the /proc file system.
+ * The returned filename points to the final executable loaded by the execve()
+ * system call. In the case of scripts, the filename points to the script
+ * handler, not to the script.
+ * The filename returned points to the actual exectuable and not a symlink.
+ *
+ */
+char* getexename(char *buf, std::size_t size)
+{
+#ifdef NO_PROC_SELF_DIR
+    char linkname[64]; // /proc/<pid>/exe
+
+    // Get our PID and build the name of the link in /proc
+    pid_t pid = ::getpid();
+
+    if (snprintf(linkname, sizeof(linkname), "/proc/%i/exe", pid) < 0)
+    {
+        //This should only happen on large word systems. I'm not sure
```

Comment:
@zeule:  I thought qBittorrent's coding style requires a space between `//` and the actual text of comments?  

Lines `444`-`447` do not follow this convention.